src/pages/Captures.tsx

```diff
@@ -1,14 +1,15 @@
 import AddIcon from '@mui/icons-material/Add';
-import { Button, Toolbar } from '@mui/material';
-import CatalogList from 'components/CatalogList';
+import { Box, Button, SxProps, Theme, Toolbar } from '@mui/material';
 import PageContainer from 'components/shared/PageContainer';
-import { useState } from 'react';
+import CapturesTable from 'components/tables/Captures';
 import { NavLink } from 'react-router-dom';
 
-const Capture = () => {
-    const [captureList] = useState([]);
-    const [isLoading] = useState(false);
+const boxStyling: SxProps<Theme> = {
+    marginBottom: 2,
```

Comment:
I use the shorthand names for these properties. Might be worthwhile to adopt that practice universally.